WebRoot/css/dwt.css

```diff
@@ -556,6 +556,10 @@ TD.DwtListView-ColumnActive {
 	@ListColHeaderText-hover@
 }
 
+/* Mail List header specific class*/
```

Comment:
should this be moved to zm.css?

Reply:
I was confused a bit, the change is for the widget (DwtListView) but the class is specific for Zimbra Mail App. Kept it here with other DwtListView styles. Should we have it in zm.css ?

Reply:
no issues in putting here as well